lib/ruby_indexer/test/configuration_test.rb

```diff
@@ -20,6 +20,23 @@ def test_load_configuration_executes_configure_block
       assert(indexables.none? { |indexable| indexable.full_path == __FILE__ })
     end
 
+    def test_supports_older_index_configuration
+      FileUtils.mv(".ruby-lsp.yml", ".ruby-lsp.yml.tmp")
```

Comment:
I could have made this just renamed `.ruby-lsp.yml` to `.index.yml` but in future they will be different structures.